pkg/mic/mictest/mic_test.go

```diff
@@ -25,36 +25,36 @@ func TestMapMICClient(t *testing.T) {
 
 	id.Name = "test-akssvcrg-id"
 	idList = append(idList, *id)
-
-	idMap, _ := micClient.ConvertIDListToMap(&idList)
-
-	name := "test-azure-identity"
-	count := 3
-	if azureID, idPresent := idMap[name]; idPresent {
-		if azureID.Name != name {
-			panic("id map id value mismatch")
+	/*
```

Comment:
commented test ?

Reply:
Yeah was in the process of refactoring. Now tests are back (except the ones related to events)